src/DonationForms/Actions/ConvertDonationFormBlocksToFieldsApi.php

```diff
@@ -175,14 +175,23 @@ protected function createNodeFromBlockWithUniqueAttributes(BlockModel $block, in
                             if (!is_user_logged_in()) {
                                 $field->required();
                             }
-                            
+
                             $field->rules(new AuthenticationRule());
                         }
                     });
 
             default:
+                $customField = apply_filters(
+                    'givewp_donation_form_block_render',
```

Comment:
@JasonTheAdams how are you anticipating using this generic filter vs the block name one? to return bulk fields or something?

Reply:
My preference is to always have a generic hook option in addition to specific hooks. My specific use-case was to be able to have a single hook function in FFM so all the block conversions can happen in a single file.